client/src/components/layout/TopBar.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import SignOutButton from "../authentication/SignOutButton";
+import "../../assets/scss/main.scss"
```

Comment:
Don't need to import into other components that are children of the router in App.js